docs/fsharp/language-reference/arrays.md

```diff
@@ -5,9 +5,6 @@ ms.date: 05/16/2016
 ---
 # Arrays
 
-> [!NOTE]
-> The API reference link will take you to MSDN.  The docs.microsoft.com API reference is not complete.
```

Comment:
The links doesn't even take to MSDN, they take to docs.microsoft.com with 404.

I think the note may be updated instead of completely deleted if there is a known issue with F# API reference?

Reply:
@cartermp It seems like you're aware of the broken links. Since the links no longer go to MSDN (they redirect to docs, but 404), I removed the notes. Should I temporarily add a reference to https://fsharp.github.io/fsharp-core-docs/?

Reply:
Yeah, I think that's a good alternative for now.